tools/query_sloop.c

```diff
@@ -1,22 +1,12 @@
-#include <acme.h>
-static inherit AcmeShadow;
-#include AcmeShadowInc
-
-
-int valid_query_sloop( object ob )
-{
-
-}
-
-object query_sloop( object victim )
-{
-    if( find_shadows( victim, "/usr/gon/tools/sloop_shad")
-    {
-        tell_object( THISP, strformat( "Sloop-a-doop!"));)
-    }
-}
-
-object *find_shadows( object host, string progname )
-{
-
-}
+// Query if player has sloop shadow on them
+// Gon
+// 2017 - 04
+
+// Declared Functions
+int query_sloop();
+
+// Find if target has sloop shadow and return 1
+int query_sloop( object victim ) {
```

Comment:
LOL! I like that. That makes sense. I was getting too focused on being similar to snoop functions, which was really unnecessary since my code is my own and I'm not truly replicating being snooped.